nbs/15_intersections/py/03-gtex_whole_blood-intersections.py

```diff
@@ -37,12 +37,12 @@
 # %% [markdown] tags=[]
 # # Settings
 
-# %%
+# %% tags=[]
```

Comment:
What's the point of the "tags=[]" ? Did something update and decide every block of code needed to have that, or did you do it for a reason?

Reply:
Oh, that's [jupytext](https://github.com/mwouts/jupytext). When it syncronizes the ipynb file with a script representation it saves some properties of each cell into the py file, such as "tags".